rust/connlib/tunnel/src/dns.rs

```diff
@@ -290,29 +279,22 @@ impl StubResolver {
         tracing::trace!("Parsed packet as DNS query: '{question}'");
 
         if let Some(records) = self.known_hosts.get_records(&question) {
-            let response =
-                build_dns_with_answer(message, question.qname(), question.qtype(), records)?;
+            let response = build_dns_with_answer(message, question.qname(), records)?;
             return Some(ResolveStrategy::LocalResponse(build_response(
                 packet, response,
             )));
         }
 
-        if !self.is_resource(&question) {
+        let Some(resource_records) = self.get_records(&question) else {
```

Comment:
Where do we now forward queries for non-resources?

Reply:
oops, pushed a fix